harvardcards/static/js/modules/collections-index.js

```diff
@@ -1,11 +1,30 @@
-define(['jquery', 'components/slider/CollectionSlider', 'utils/utils'], function($, CollectionSlider, utils) {
+define(['jquery', 'components/slider/CollectionSlider', 'utils/utils', 'jqueryui'], function($, CollectionSlider, utils) {
 	return {
 		initModule: function(){
-			$('.slider').each(function(index, el) {
-				var slider = new CollectionSlider(el);
-			});
 
-			utils.setupConfirm();
+
+            $('.courseHeader').click(function() {
+                var el = this;
+                $(this).next().toggle('slow', function(){
+                    var collection_id = $(el).data('collection-id');
+                    if($(el).next().is(":visible")){
+                        localStorage.setItem('show-collection-id'+collection_id, true);
```

Comment:
Is there any advantage to using localStorage like this:

```javascript
localStorage.setItem('show-collection-id' + collection_id, bool);```
```

Instead of this?

```javascript
dict = {};
dict[collection_id] = true; # true | false
localStorage.setItem('collections-visible', dict);
```

The former seems fine to me, just wondering.